tasks/thp.yml

```diff
@@ -0,0 +1,15 @@
+---
+
+- name: delete Transparent Huge Pages service file
+  become: true
+  file:
+    path: /usr/lib/systemd/system/disable-transparent-huge-pages.service
+    state: absent
+
+- name: disable Transparent Huge Pages flag via grub
```

Comment:
This should probably be configurable/skippable, as LXC-deployments don't have GRUB installed, which causes this task to fail